EXOSIMS/Prototypes/SimulatedUniverse.py

```diff
@@ -708,9 +708,15 @@ def load_systems(self, systems):
         self.p = systems["p"]
         self.plan2star = systems["plan2star"]
 
-        if "starI" in systems:
-            self.TargetList.I = systems["starI"]  # noqa: E741
-            self.commonSystemInclinations = True
+        if "systemInclination" in systems:
+            self.TargetList.systemInclination = systems[
+                "systemInclination"
+            ]  # noqa: E741

```

Comment:
The #noqa shouldn't be necessary anymore in this case (since you changed the var name from I).